setup.md

```diff
@@ -98,6 +98,16 @@ pip install tensorflow>=2.5 seaborn scikit-learn pandas
 [pip](https://pip.pypa.io/en/stable/) is the package management system for Python software packages.
 It is integrated into your local Python installation and runs regardless of your operating system too.
 
+### Troubleshooting for Macs with Apple M1 chip
+Newer Macs (from 2020 onwards) often have an Apple M1 chip instead of Intel.
```

Comment:
```suggestion
Newer Macs (from 2020 onwards) often have a different kind of chip, manufactured by Apple instead of Intel.
```

I recommend avoiding referring to M1 specifically. I think M2s are already in production and other numbers will presumably appear in the future as well.